marimbabot_sim/urdf/marimba.urdf.xacro

```diff
@@ -398,4 +398,11 @@ TODO:
 <!-- TODO instantiate in standalone files -->
 <xacro:marimba name="marimba" />
 
+<link name="world"/>
```

Comment:
I don't know if we want to define the position of the marimba in the world statically? It likely needs to be calibrated quite often / estimated via an April tag.